src/main/java/com/stripe/model/SubscriptionSchedule.java

```diff
@@ -0,0 +1,406 @@
+// Generated by com.stripe.generator.entity.SdkBuilder
+
+package com.stripe.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.stripe.Stripe;
+import com.stripe.exception.StripeException;
+import com.stripe.net.ApiResource;
+import com.stripe.net.RequestOptions;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = false)
+public class SubscriptionSchedule extends ApiResource
+    implements HasId, MetadataStore<SubscriptionSchedule> {
+  /**
+   * Either `charge_automatically`, or `send_invoice`. When charging automatically, Stripe will
+   * attempt to pay the underlying subscription at the end of each billing cycle using the default
+   * source attached to the customer. When sending an invoice, Stripe will email your customer an
+   * invoice with payment instructions.
+   */
+  @SerializedName("billing")
+  String billing;
+
+  /**
+   * Define thresholds at which an invoice will be sent, and the subscription advanced to a new
+   * billing period.
+   */
+  @SerializedName("billing_thresholds")
+  Subscription.BillingThresholds billingThresholds;
+
+  /**
+   * Time at which the subscription schedule was canceled. Measured in seconds since the Unix epoch.
+   */
+  @SerializedName("canceled_at")
+  Long canceledAt;
+
+  /**
+   * Time at which the subscription schedule was completed. Measured in seconds since the Unix
+   * epoch.
+   */
+  @SerializedName("completed_at")
+  Long completedAt;
+
+  /** Time at which the object was created. Measured in seconds since the Unix epoch. */
+  @SerializedName("created")
+  Long created;
+
+  /**
+   * Object representing the start and end dates for the current phase of the subscription schedule,
+   * if it is `active`.
+   */
+  @SerializedName("current_phase")
+  CurrentPhase currentPhase;
+
+  /** ID of the customer who owns the subscription schedule. */
+  @SerializedName("customer")
+  @Getter(lombok.AccessLevel.NONE)
+  @Setter(lombok.AccessLevel.NONE)
+  ExpandableField<Customer> customer;
+
+  /** Unique identifier for the object. */
+  @Getter(onMethod = @__({@Override}))
+  @SerializedName("id")
+  String id;
+
+  /** The subscription schedule's default invoice settings. */
+  @SerializedName("invoice_settings")
+  SubscriptionScheduleInvoiceSettings invoiceSettings;
```

Comment:
This is one of the examples where we are "nesting it" but really it's likely that in 4 months we share that hash with another resource and so maybe it should be top-level.

I gave up for .NET and go and did what you did (but not nested because those languages can't)